docusaurus/docs/recipes/favicon-os-theme.md

```diff
@@ -0,0 +1,55 @@
+---
+id: favicon-os-theme
+title: Changing favicon according to OS theme
+sidebar_label: Changing favicon according to OS theme
+---
+
+Most operating systems offer by now light and dark modes. Sometimes your favicon might not be visible in one of the themes.
+
+This recipe explains how to use different favicons for different OS themes. To do so follow the next steps:
+
+1. Install the package.
+
+   `npm install react-use-system-theme`
+
+2. Use the `useSystemTheme` hook in the `App.js` file to get the current OS theme.
+
+   You can use `dark` or `light` as a parameter to setup a default theme. Find more info [here](https://github.com/zebateira/react-use-system-theme).
+
+3. Add the new favicon files to the `public/favicons` folder.
+
+4. Add conditional clauses to the HTML.
+   If your default favicon works better with a light background you should use `dark` theme for the conditional clauses or vice versa.
+
+Example of the `App.js` file:
```

Comment:
Shouldn’t this be indebted too as well as the code below?

Reply:
I didn't do it because the example code is a result of all the instructions above, especially points 2 and 4.